webapp/app/viewModels/role/RoleAdd.js

```diff
@@ -0,0 +1,23 @@
+require('services/roleService');
+
+define(['services/roleService'], function (roleService) {
+    
+    var vm = {
+        getView: function () {
+            var view = require('./roleAdd.html');
+            return $.parseHTML(view)[0];
+        },
+        name: ko.observable(),
+        roleAdd: roleAdd,
+        roleResponse: ko.observable('')
+    };
+
+    function roleAdd() {
+        roleService.roleAdd(this.name()).then(function (response) {  
+            var url = '#roleList/';
```

Comment:
Same here: Do you need this auxiliary variable?

Reply:
i remove the var url